lib/rspec-console/runner.rb

```diff
@@ -1,44 +1,16 @@
+# This class wraps the core rspec runner and manages the environment around it.
 class RSpecConsole::Runner
   class << self
-    def reset(args)
-      require 'rspec/core'
-
-      if Gem.loaded_specs['rspec-core'].version < Gem::Version.new('2.9.10')
-        raise 'Please use RSpec 2.9.10 or later'
-      end
-
-      ::RSpec::Core::Runner.disable_autorun!
-      ::RSpec::Core::Configuration.class_eval { define_method(:command) { 'rspec' } }
-      ::RSpec.reset
-
-      config_cache.cache do
-        ::RSpec.configure do |config|
-          config.output_stream = STDOUT
-          config.color_enabled = true if config.respond_to?(:color_enabled=)
-          config.color         = true if config.respond_to?(:color=)
-        end
-
-        $LOAD_PATH << './spec'
-        require "spec_helper"
-        begin
-          require "rails_helper"
-        rescue LoadError
-        end
-      end
-    end
-
     def run(args)
       RSpecConsole.hooks.each(&:call)
-      reset(args)
+
+      RSpecConsole::Environment.reset
```

Comment:
sorry no, you are right. It's just that I got tripped over because `reset` does either the recording, and then subsequently the replay. So to me, the reset means the replay side.
`prepare_rspec_config` would be a little more meaningful. Or maybe it's the wording `Environment`, it could mean anything. `RSpecConfigurator` would be more appropriate perhaps.